biglog/biglog.go

```diff
@@ -249,15 +263,18 @@ func (bl *BigLog) latest() (o int64) {
 	return absolute(hotSeg.NRO-1, hotSeg.baseOffset)
 }
 
-// Name returns the big log's name, which maps to the folder name
-func (bl *BigLog) Name() (s string) {
+// Name returns the big log's name, which maps to directory path that contains
+// the index and data files.
+func (bl *BigLog) Name() string {
 	bl.mu.RLock()
 	defer bl.mu.RUnlock()
 	return bl.name
 }
 
-// Split generates a new segment recipient of future writes
-func (bl *BigLog) Split() (err error) {
+// Split creates a new segment in bl's dirPath using the maxIndexEntries of
+// the currently active segment starting at the highest available offset+1.
```

Comment:
how about this?
```
// Split creates a new segment in bl's dirPath starting at the highest
// available offset+1. The new segment has the same size as the old one
// and becomes the new hot (active) segment.
```

Alternatively its probably also ok here to omit the segment size altogether. This kind of information could be documented in the high level package documentation describing the file layout of a Biglog.